lib/pagseguro.rb

```diff
@@ -6,12 +6,15 @@
 require "aitch"
 require "i18n"
 
+require "pagseguro/extensions/mass_assignment"
+require "pagseguro/extensions/ensure_type"
+
 require "pagseguro/version"
 require "pagseguro/config"
```

Comment:
just a small detail here. maybe, it's better to require version and config before extensions.